envbuilder.go

```diff
@@ -111,6 +111,10 @@ type Options struct {
 	// It will override CacheRepo if both are specified.
 	LayerCacheDir string `env:"LAYER_CACHE_DIR"`
 
+	// CacheTTLDays is the number of days to use cached layers before
+	// expiring them. Defaults to 7 days.
+	CacheTTLDays int `env:"CACHE_TTL_DAYS"`
```

Comment:
Should we make this a `time.Duration` instead? Go can parse pretty nice strings like "7d" or "12hr".

Reply:
I was thinking about that, but since days aren't supported in the format, it's an awkward way to specify long time periods. Okay with me if you prefer that.